modules/mssql/variables.tf

```diff
@@ -306,3 +306,9 @@ variable "deletion_protection" {
   type        = bool
   default     = true
 }
+
+variable "time_zone" {
+  description = "The time zone for SQL instance."
+  type        = string
+  default     = "UTC"
```

Comment:
nit: can we set this to null so we use whatever is provider default?

Reply:
> nit: can we set this to null so we use whatever is provider default?

Nice catch, it was a bit of an overlook on my end, but fixed now. Thanks @bharathkkb